griptape/labs/tools/__init__.py

```diff
@@ -6,7 +6,7 @@
 from .aws_s3.tool import AwsS3
 from .zoom.tool import Zoom
 from .openai_dalle.tool import OpenAiDallE
-from .sql_redshift.tool import SqlRedshiftClient
+from .aws_redshift.tool import AwsRedshiftClient
```

Comment:
Let's call this package `aws_redshift_client`, so it's consistent with other package names.